var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -228,54 +228,55 @@ def install(self, spec, prefix):
 
         # take step-40 which can use both PETSc and Trilinos
         # FIXME: switch step-40 to MPI run
-        with working_dir('examples/step-40'):
-            print('=====================================')
-            print('========== Step-40 PETSc ============')
-            print('=====================================')
-            # list the number of cycles to speed up
-            filter_file(r'(const unsigned int n_cycles = 8;)',
-                        ('const unsigned int n_cycles = 2;'), 'step-40.cc')
-            cmake('.')
-            if '^petsc' in spec:
-                make('release')
-                make('run', parallel=False)
-
-            print('=====================================')
-            print('========= Step-40 Trilinos ==========')
-            print('=====================================')
-            # change Linear Algebra to Trilinos
-            filter_file(r'(\/\/ #define FORCE_USE_OF_TRILINOS.*)',
```

Comment:
p.s. contrary to the comments below, i think it is good to run an example which use Trilinos. So in this instance i would add another version of `filter_file` to comply with `8.3.0`.